web/src/presence.ts

```diff
@@ -187,7 +216,7 @@ export function set_info(presences, server_timestamp) {
             continue;
         }
 
-        const raw = {
+        const raw: RawPresence = {
             server_timestamp,
             active_timestamp: info.active_timestamp || undefined,
             idle_timestamp: info.idle_timestamp || undefined,
```

Comment:
I'm not entirely sure, but why do we need the `|| undefined` fallback if `active_timestamp` is known to be `number | undefined`?

I understand that a truthiness check might be desired, but is it possible for any of these to be `0`? One more thing is that we are having similar fallbacks of the form `|| 0` elsewhere in this file that are sort of doing the reverse of this check here.

Reply:
Yeah I think there may be a potential refactoring we can do here, I will look for why we have these undefined values lying all over the place in this module as a follow-up.